persistence-cassandra-3.11/README.md

```diff
@@ -0,0 +1,27 @@
+# Persistence Cassandra 3.11
+
+This module represents the implementation of the [persistence-api](../persistence-api) for the Cassandra `3.11.x` version.
+
+## Cassandra version update
+
+The current Cassandra version this module depends on is `3.11.11`.
+In order to update to a newer patch version, please follow the guidelines below:
+
+* Update the `cassandra.version` property in the [pom.xml](pom.xml).
+* Check what is the version of the `com.datastax.cassandra:cassandra-driver-core` in the `org.apache.cassandra:cassandra-all` for the updated version. 
```

Comment:
I don't understand what this means. Is it saying to look at a setting in a Maven repo or the Apache Cassandra codebase?

Reply:
Transitive dependencies can be seen in the pom.xml, on the web links like https://mvnrepository.com/artifact/org.apache.cassandra/cassandra-all/3.11.11, or using `./mvnw dependency:tree -pl persistence-cassandra-3.11`.. Should I define the explicit way to do it? What would you like the best? Any proposal for rephrasing it?

Reply:
Added more detailed explanation. Considering done.